README.md

```diff
@@ -1,7 +1,45 @@
 [![Build Status](https://travis-ci.org/dedis/onet.svg?branch=master)](https://travis-ci.org/dedis/onet)
-[![Coverage Status](https://coveralls.io/repos/github/dedis/onet/badge.svg?branch=travis_24)](https://coveralls.io/github/dedis/onet?branch=travis_24)
+[![Coverage Status](https://coveralls.io/repos/github/dedis/onet/badge.svg)](https://coveralls.io/github/dedis/onet)
 
-# onet
+# The Cothority Network Library - CoNet
```

Comment:
So we're waiting to see the final organization and naming on next thursday to see how it's called ? conet vs onet vs core ?
Otherwise, let's start calling it core ?